_includes/manuals/1.24/1.2_release_notes.md

```diff
@@ -2,7 +2,7 @@
 
 ### Headline features
 #### Support for Azure Resource Manager Compute Resource
-Foreman can now provision hosts using Azure Resource Manager (AzureRM) as a Compute Resource via the `foreman_azure_rm` plugin. Provisioning can be successfully done on Linux platforms with either Finish template or Userdata template. Currently, it supports single OS disk for the VM and uses public images provided by Azure Marketplace.
+Foreman can now provision hosts using Azure Resource Manager (AzureRM) as a Compute Resource via the `foreman_azure_rm` plugin. Provisioning can be successfully done on Linux platforms with either Finish template or Userdata template. Currently, it supports single OS disk for the VM and uses public images provided by Azure Marketplace. For more information, head over to [Foreman Azure](https://theforeman.org/plugins/foreman_azure/2.0.x/index.html) manual.
```

Comment:
@tbrisker Did you mean adding the manual's link here in the headline's section, like this?

Reply:
exactly! :1st_place_medal: